### ISSUE TITLE
Unzip issues on i2pseeds.su3 file

### ISSUE THREAD

**Issue by wqking** [monero-project/kovri#795]
OS: Win10. Kovri was built using MSys2, MinGW64.

I download the seeds file here,
https://i2p.novg.net/i2pseeds.su3
I also attached the file.
[i2pseeds.su3.zip](https://github.com/monero-project/kovri/files/1662682/i2pseeds.su3.zip)

**Issue 1:**
Command
`kovri.exe --reseed-from i2pseeds.su3`
The memory usage jump between 1G and 4G very quickly, and after some long time, kovri gives some zip related error.

**Issue 2**
Command
`kovri.exe --enable-su3-verification=0 --reseed-from i2pseeds.su3`
kovri gives zip related error right away. `[error]   ZIP: CRC-32 Failed`

I also use WinRar to unzip the .su3, then compress it using standard zip method, both issues 1 & 2 exist.

My thoughts: Kovri uses its own version unzip code, seems it doesn't support the zip standard well.
If the issue is confirmed, I'm happy to work on it. We may replace the zip module with existing mature zip library, etc.

---
**By submitting this issue, I confirm the following:**

- I have read and understood the contributor guide in [kovri-docs](https://github.com/monero-project/kovri-docs).
- I have checked that the issue I am reporting can be replicated or that the feature I am suggesting is not present.
- I have checked opened or recently closed [pull requests](https://github.com/monero-project/kovri/pulls) for existing solutions/implementations to my issue/suggestion.
---

**Comment by wqking** [comment]
And refactor/redesign/rewrite the unzip part may help  #785

**Comment by anonimal** [comment]
I cannot reproduce either issue using the same su3 file. Please attach a log with `--log-level 5`.

**Comment by anonimal** [comment]
>And refactor/redesign/rewrite the unzip part may help #785

This issue is completely unrelated to #785. #785 is simply about undoing the pimpl work that was done.

**Comment by anonimal** [comment]
@wqking can you please upload the following:

1. Log output with `--log-level 5`
2. Complete build information

Thank you.

**Comment by wqking** [comment]
[kovri-build.txt](https://github.com/monero-project/kovri/files/1676248/kovri-build.txt)
[kovri-log.txt](https://github.com/monero-project/kovri/files/1676249/kovri-log.txt)

I uploaded the log files. kovri-build.txt is the build information, kovri-log.txt is the log.
My build environment is MSys2 MinGW64. I used it to build Monero successfully so I think it should be fine.
The pre-built binary for Windows I downloaded from the official site also crashes on my Win10. I guess for the crash issues, as well as the unzip issues, is caused by possible something on my Win10 is not quite friendly to Kovri though I'm not sure. These days I don't have time yet, when I have, I may debug to see what's wrong there, especially what's wrong with the unzip function.

**Comment by anonimal** [comment]
>kovri-build.txt

Please build against current master or d6a48512622387d67f339ad5e6876afafe6ebdd3, whichever is newer by the time you read this. Also, the file doesn't show complete build information. Please paste *complete* build information as well as `$ uname -a`.

>kovri-log.txt

This doesn't show complete log information. Please paste *complete* log information from router start to end.

The su3 file you're using is now stale. If you can, please grab a fresh one from one of the hardcoded reseed servers that is *not* https://i2p.novg.net/, using the following command or user-agent: `$ wget -c --user-agent="Wget/1.11.4"`, then upload the su3.

**Comment by wqking** [comment]
I built Kovri on Ubuntu 16.0.4 (VirtualBox on my Win10) and it doesn't have any error with the su3 file.
So I guess the problem I reported here is caused by either my Win10 MSys2 environment or the compatibility problem of Kovri on Windows.
I suggest that we put this issue either low priority or just close it, though we need a thorough compatibility test/fix for Windows at the final stage. I've setup my Ubuntu box seriously and gonna switch to Ubuntu development rather than Windows to make my life easier in open source projects.
